Dashboard/app/js/lib/controllers/permissions.js

```diff
@@ -446,7 +446,7 @@ FLOW.dialogControl = Ember.Object.create({
 
     case "delForm":
```

Comment:
After reviewing, I believe that actually there is a simpler fix for this.  in this same switch statement, the case `delS` seems to have been previously applying to the deletion of a form.  This case has code that actually performs the preflight check and then only accepts to delete if the preflight check returns correct. take a look there and see what you can figure out